content/docs/started/getting-started-gke.md

```diff
@@ -72,8 +72,15 @@ Create an OAuth client ID to be used to identify Cloud IAP when requesting acces
     export CLIENT_ID=<CLIENT_ID from OAuth page>
     export CLIENT_SECRET=<CLIENT_SECRET from OAuth page>
     ```
+## Deploy Kubeflow on Kubernetes Engine Using The Ui
```

Comment:
Fix capitalisation of "UI" and use sentence case instead of title case for rest of heading:

Deploy Kubeflow on Kubernetes Engine using the UI